packages/collector/test/test_util/TeeLogger.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ */
+
+'use strict';
+
+/**
+ * Multiplexes logs to a collection of target loggers. Think tee (Like https://man7.org/linux/man-pages/man1/tee.1.html)
+ * but for loggers.
+ */
+
+module.exports = class TeeLogger {
```

Comment:
This is used to also still see logs on the console/in the application's output while also collecting the logs for inspection in the test.